config/environments/production.rb

```diff
@@ -39,16 +38,22 @@
   # config.action_cable.url = "wss://example.com/cable"
   # config.action_cable.allowed_request_origins = [ "http://example.com", /http:\/\/example.*/ ]
 
+  # Assume all access to the app is happening through a SSL-terminating reverse proxy.
+  # Can be used together with config.force_ssl for Strict-Transport-Security and secure cookies.
+  config.assume_ssl = true
```

Comment:
```suggestion
  config.assume_ssl = ENV.fetch('RAILS_ASSUME_SSL', 'true') == 'true'
```

Reply:
You okay to update Makefile to include https://github.com/trade-tariff/trade-tariff-frontend/blob/main/Makefile#L8